docs/README.md

```
@@ -240,6 +240,52 @@ cd training_hub
 uv pip install -e . && uv pip install -e .[cuda] --no-build-isolation
 ```
 
+## Coding Agent Plugin
+
+Training Hub is available as a plugin for two coding agents, bringing LLM training capabilities directly into your coding workflow.
+
+<details>
+<summary><strong>Claude Code</strong></summary>
+
+**Via org marketplace** (recommended — includes all Red Hat AI plugins):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)